staging/src/k8s.io/kubectl/pkg/cmd/debug/debug.go

```diff
@@ -160,7 +160,7 @@ func addDebugFlags(cmd *cobra.Command, opt *DebugOptions) {
 	cmd.Flags().StringToString("env", nil, i18n.T("Environment variables to set in the container."))
 	cmd.Flags().StringVar(&opt.Image, "image", opt.Image, i18n.T("Container image to use for debug container."))
 	cmd.MarkFlagRequired("image")
-	cmd.Flags().String("image-pull-policy", string(corev1.PullIfNotPresent), i18n.T("The image pull policy for the container."))
+	cmd.Flags().String("image-pull-policy", "", i18n.T("The image pull policy for the container."))
```

Comment:
Could you add a message to the description about the default behavior?

Reply:
done.